### PR TITLE
[Tizen Studio] 4.0 release notes

### PR DESCRIPTION
**Pull request by manishtr**
### Change Description ###

Added Tizen Studio 3.8 release notes.

Signed-off-by: manish.r <manish.r@samsung.com>

### CODE REVIEW

File: docs/application/tizen-studio/release-notes/3-8-release-notes.md
Comment (manishtr) [reply docs/application/tizen-studio/release-notes/3-8-release-notes.md:10]: updated

File: docs/application/tizen-studio/release-notes/3-8-release-notes.md
Comment (manishtr) [reply docs/application/tizen-studio/release-notes/3-8-release-notes.md:11]: updated

File: docs/application/tizen-studio/release-notes/3-8-release-notes.md
Comment (manishtr) [reply docs/application/tizen-studio/release-notes/3-8-release-notes.md:25]: Updated.

File: docs/application/tizen-studio/release-notes/3-8-release-notes.md
Comment (manishtr) [reply docs/application/tizen-studio/release-notes/3-8-release-notes.md:41]: updated

**Comment by mijin85cho** [comment]
Thanks for the update and review.

As the target date to release Tizen Studio 3.8 is "June 30",
I'll postpone the merge of this PR to that date, to prevent exposing this release note before the release date.

**Comment by nawab-samsung** [comment]
@mijin85cho sure & thanks.

**Comment by manishtr** [comment]
> Thanks for the update and review.
> 
> As the target date to release Tizen Studio 3.8 is "June 30",
> I'll postpone the merge of this PR to that date, to prevent exposing this release note before the release date.

Thanks for the review, i will update incase there is a change in the release date.

**Comment by manishtr** [comment]
> > Thanks for the update and review.
> > As the target date to release Tizen Studio 3.8 is "June 30",
> > I'll postpone the merge of this PR to that date, to prevent exposing this release note before the release date.
> 
> Thanks for the review, i will update incase there is a change in the release date.

updated the release date to july 10, remoed one issue as it has been fixed

**Comment by mijin85cho** [comment]
> > > Thanks for the update and review.
> > > As the target date to release Tizen Studio 3.8 is "June 30",
> > > I'll postpone the merge of this PR to that date, to prevent exposing this release note before the release date.
> > 
> > 
> > Thanks for the review, i will update incase there is a change in the release date.
> 
> updated the release date to july 10, remoed one issue as it has been fixed

With this notification, I got the information that the Tizen Studio 3.8 release is postponed to (approximately) 7.30.
I'll keep holding on this PR till that date.

**Comment by mijin85cho** [comment]
FYI. I got notification from @sangwook96 that the release of Tizen Studio 3.8 is postponed.
The release is now targeting to be done in August.

File: docs/application/toc_all.md
Comment (mijin85cho) [inline docs/application/toc_all.md:1180]:
```suggestion
### [4.0 (Oct 31, 2020)](/application/tizen-studio/release-notes/4-0-release-notes.md)
```

File: docs/application/tizen-studio/release-notes/4-0-release-notes.md
Comment (mijin85cho) [inline docs/application/tizen-studio/release-notes/4-0-release-notes.md:3]:
```suggestion
- Release Date: Oct 31, 2020
```

**Review by mijin85cho — COMMENTED** [review]
Hello, @nawab-samsung 
The release note was updated since the last review.
So, we need to review it again, and I think reviewing  the only updates parts will be enough, regarding https://github.com/Samsung/tizen-docs/pull/1140/commits/4a7e207a76bc59f5f5daad1aa46a522c28f3cbb6.

Would you please check the latest and leave your comments?

**Comment by mijin85cho** [comment]
FYI : This PR is requested to finish the review before Oct 21.
@nawab-samsung would you please leave the comments ASAP so that we can "approve" this PR till Wednesday?

**Review by mijin85cho — COMMENTED** [review]
Below are my queries about the added part since the approved version. Would you please check?

Also @nawab-samsung, 
to finish the review of this PR till tomorrow, would you please give this PR high priority?
The parts where need review are the added sentences by https://github.com/Samsung/tizen-docs/pull/1140/commits/4a7e207a76bc59f5f5daad1aa46a522c28f3cbb6 and https://github.com/Samsung/tizen-docs/pull/1140/commits/3d2227c8d90c9dc149a76775c165f14847cbe619.

File: docs/application/tizen-studio/release-notes/4-0-release-notes.md
Comment (manishtr) [reply docs/application/tizen-studio/release-notes/4-0-release-notes.md:19]: Currently it not decided as the next release might only contain important bug fixes and 5.5 Updates

File: docs/application/tizen-studio/release-notes/4-0-release-notes.md
Comment (nawab-samsung) [inline docs/application/tizen-studio/release-notes/4-0-release-notes.md:18]: @mijin85cho what's your opinion? 
changing starting with from // just to maintain consistency with other notes.
```suggestion 
   - Tizen RT will be deprecated from Tizen Studio 4.1.

File: docs/application/tizen-studio/release-notes/4-0-release-notes.md
Comment (nawab-samsung) [inline docs/application/tizen-studio/release-notes/4-0-release-notes.md:27]:
```suggestion
  - If you install Tizen Studio in a directory that requires administrator privileges for access. For example, for **C:\ProgramFiles** location administrator privilege is needed to run the Tizen SDK tools. The Tizen Installer and the baseline SDK Installer alerts you if you try to install it into such a directory.

File: docs/application/tizen-studio/release-notes/4-0-release-notes.md
Comment (nawab-samsung) [inline docs/application/tizen-studio/release-notes/4-0-release-notes.md:29]:
```suggestion 
  - You can create unit tests for Tizen 2.3.2 and higher version projects only. Unit testing for older versions is not supported.

File: docs/application/tizen-studio/release-notes/4-0-release-notes.md
Comment (nawab-samsung) [inline docs/application/tizen-studio/release-notes/4-0-release-notes.md:46]:
```suggestion
  - If you use a MacBook Pro that has both Intel HD and NVIDIA&reg; GPUs, the emulator can terminate unexpectedly when you execute the emulator with OpenGL ES version 1.1 or 2.0. Verify the emulator configuration in Emulator Manager and on the **General** tab in the emulator configuration window, set OpenGL ES version to version 2.0, or to version 3.0.

File: docs/application/tizen-studio/release-notes/4-0-release-notes.md
Comment (nawab-samsung) [inline docs/application/tizen-studio/release-notes/4-0-release-notes.md:49]:
```suggestion
  - To use Tizen Emulator in Tizen platform 3.0 or lower, disable the CPU VT option in the **HW Support** tab of Emulator Configuration.

**Review by nawab-samsung — CHANGES_REQUESTED** [review]
Request a few changes,  free to ask queries.

File: docs/application/tizen-studio/release-notes/4-0-release-notes.md
Comment (nawab-samsung) [reply docs/application/tizen-studio/release-notes/4-0-release-notes.md:19]: @manishtr can you please indent the 2nd point of the list. 

No other changes needed, following which I shall approve the PR.
Thanks

File: docs/application/tizen-studio/release-notes/4-0-release-notes.md
Comment (nawab-samsung) [inline docs/application/tizen-studio/release-notes/4-0-release-notes.md:19]:
```suggestion
   - Tizen RT will be deprecated from Tizen Studio 4.1.
   - Bridge Networking support in emulators will be deprecated in future Tizen Studio releases.

**Review by mijin85cho — COMMENTED** [review]
As @manishtr updated the files according to your comments, @nawab-samsung, we are waiting your reply.

File: docs/application/tizen-studio/release-notes/4-0-release-notes.md
Comment (manishtr) [reply docs/application/tizen-studio/release-notes/4-0-release-notes.md:37]: This sentence is about 5.5 Version , it has this issue, the patch for it will be release with 4.1 or later.

Comment (mijin85cho) [reply docs/application/tizen-studio/release-notes/4-0-release-notes.md:37]: Thanks for the reply, @manishtr. Then we can keep the sentence.

**Comment by mijin85cho** [comment]
@nawab-samsung Would you please check the latest version and approve the PR if you see no more issues?

FYI : As the release date is Oct 31, I'll hold this PR till the actual release date even it is approved.

**Comment by nawab-samsung** [comment]
@mijin85cho sure

File: docs/application/tizen-studio/release-notes/4-0-release-notes.md
Comment (nawab-samsung) [inline docs/application/tizen-studio/release-notes/4-0-release-notes.md:27]: @manishtr Although the current sentences give the meaning, but I am suggesting the following:  

```suggestion
  - If you install Tizen Studio in a directory such as **C:\ProgramFiles** that needs administrator privileges to run the Tizen SDK tools.  The Tizen Installer and the baseline SDK Installer alerts you.
```
If you like it, then incorporate, else just ignore.

Comment (mijin85cho) [reply docs/application/tizen-studio/release-notes/4-0-release-notes.md:27]: @nawab-samsung As we have to finish the review of this PR till today, and the last issue you reported is minor, 
we'd better to skip this comment.

Please remember this issue to check when release note for next Tizen Studio version is created.

If there is no other critical issue except this one, approve this PR.

Comment (nawab-samsung) [reply docs/application/tizen-studio/release-notes/4-0-release-notes.md:27]: Sure, thanks

**Review by nawab-samsung — APPROVED** [review]
Thanks @manishtr 
Thanks @mijin85cho for your suggestions.

**Comment by manishtr** [comment]
Corrected a spelling mistake for the word Toolchain , sorry for the last minute update

**Review by nawab-samsung — APPROVED** [review]
Thanks @manishtr

File: docs/application/tizen-studio/release-notes/4-0-release-notes.md
Comment (mijin85cho) [inline docs/application/tizen-studio/release-notes/4-0-release-notes.md:3]: I think @manishtr would also know, that the release date is changed again.
At here it is said that Tizen Studio 4.0 is planned to be published on Oct 29.
So, would you please update the release date to the actual date?

I hope this comment be the last update request...

File: docs/application/toc_all.md
Comment (mijin85cho) [inline docs/application/toc_all.md:1180]: Please update to "4.0 (Oct 29, 2020)"

File: docs/application/tizen-studio/release-notes/4-0-release-notes.md
Comment (manishtr) [reply docs/application/tizen-studio/release-notes/4-0-release-notes.md]: Thanks, i have done it

File: docs/application/toc_all.md
Comment (manishtr) [reply docs/application/toc_all.md]: Thanks, i have done it

**Review by mijin85cho — APPROVED** [review]
I appreciate your updates.

**Pull request merged**
